tests/queensidecastle.test.ts

```diff
@@ -1,6 +1,6 @@
 import { Chess } from '../cjsmin/src/chess';
 
-xdescribe('Queenside castling is detected', () => {
+describe('Queenside castling is detected', () => {
```

Comment:
where does the .filter function come from? chess.js?

Reply:
this is old test. Filter is a method applicable to arrays